portfolio/src/main/webapp/contactMe.js

```diff
@@ -1,17 +1,13 @@
-document.addEventListener('DOMContentLoaded', renderMessages)
+renderMessages();
```

Comment:
I realized that calling it is the same than adding an event listener, since the script is loaded with the 'defer' keyword, it will run after everything is loaded